app/controllers/users.ts

```diff
@@ -47,3 +48,27 @@ export async function createUser(req: Request, res: Response, next: NextFunction
       next(databaseError);
     });
 }
+
+export async function authUser(req: Request, res: Response, next: NextFunction): Promise<Response | void> {
+  const { email, password } = req.body;
+
+  try {
+    const user: User | undefined = await userService.findUser({ email });
```

Comment:
I think that instead of saving the information in a user object it could have a loginUser interface

Reply:
I tried to create an loginUser interface but typescript doesn't allow me because the "findUser" service returns a user object.